static/src/js/components/Preferences/PreferencesRadioField.jsx

```diff
@@ -38,10 +38,27 @@ class PreferencesRadioField extends Component {
 
     const {
       enum: values,
-      enumNames,
-      description
+      description,
+      title,
+      enumNames
     } = schema
 
+    let parsedFormData = formData
+    // TODO remove this in a subsequent release
```

Comment:
Write the ticket to do this work and leave the ticket number in this TODO comment